include/cucumber-cpp/internal/defs.hpp

```diff
@@ -1,5 +1,6 @@
 #include "step/StepManager.hpp"
 #include "hook/HookRegistrar.hpp"
 #include "ContextManager.hpp"
+#include "CukeDll.hpp"
```

Comment:
Please follow standard "[include-what-you-use](https://include-what-you-use.org/)" rules and _don't_ include what you don't use (in the same file).